packages/dashboard/src/components/pageStructure/stories/menuButton.js

```diff
@@ -27,19 +22,20 @@ import NavMenuButton from '../menuButton';
 
 export default {
   title: 'Dashboard/Components/NavMenuButton',
+  args: {
+    showOnlyOnSmallViewport: true,
```

Comment:
💭 Nit: this name is difficult to understand. I know it's the prop name but maybe we could name it `Show hamburger menu icon on large screen` or something.